### DISCUSSION TITLE
Draco and WebGL

### DISCUSSION THREAD

**Discussion by aer0ace** [atteneder/glTFast#378] [Q&A] [answered]
Hello,

I'm running Unity 2020.3 LTS and I'm building a WebGL app. I am using DracoUnity 3.3.2, but the dlls don't seem to work when importing gltf files. I get the "expected x64 architecture, but was Unknown architecture" error. I'm using glTFast 4.6.0 with DracoUnity 3.3.2 when getting this result.

However, I can say that I can successfully get DracoUnity 4.x working with glTFast 4.6.0, but only for Windows builds. I am aware that WebGL builds won't compile with this combination on anything less than Unity 2021.x. Is upgrading really my only option here? I'd prefer to get DracoUnity 3.3.2 working with 2020.3 LTS though.

**Comment by atteneder** [comment, marked as answer]
@aer0ace Thanks for reporting, though this seems more like an issue and less of a discussion.

If the DLL is broken, it should be fixable. I'll try to reproduce now. You're on Windows 64-bit, I presume?

**Reply by atteneder** [reply]
@aer0ace Which Windows version btw?

**Reply by aer0ace** [reply]
Yes, Windows 64, using Unity 2020.3.32f1 LTS.

**Reply by atteneder** [reply]
It works for me in the Editor. Did you install DracoUnity 3.3.2 via OpenUPM? If you installed it via GIT, chances are you did not run GIT LFS pull and have only a partial clone.

**Reply by aer0ace** [reply]
I installed via GIT. I'll try via OpenUPM.

**Reply by aer0ace** [reply]
I ran into no problems installing via OpenUPM. Thanks for that! Another system I am using may not be able to use OpenUPM due to network policies. If that were the case, I will still need to get it via Git. I see your note in the readme regarding LFS. Thanks again!

**Reply by aer0ace** [reply]
Unfortunately, I cannot use OpenUPM due to network policies. I was able to get the proper version via github with this link though:
https://github.com/atteneder/DracoUnity.git#v3.3.2

I had realized you can append the version, and it worked fine. Thanks!
